packages/block-library/src/media-text/media-container.js

```diff
@@ -55,30 +55,18 @@ class MediaContainer extends Component {
 	}
 
 	renderImage() {
-		const { mediaAlt, mediaUrl, className } = this.props;
+		const { mediaAlt, mediaUrl, className, imageFill, focalPoint } = this.props;
+		const backgroundStyles = imageFill ? imageFillStyles( mediaUrl, focalPoint ) : {};
 		return (
 			<Fragment>
 				{ this.renderToolbarEditButton() }
-				<figure className={ className }>
+				<figure className={ className } style={ backgroundStyles }>
 					<img src={ mediaUrl } alt={ mediaAlt } />
```

Comment:
Despite what I said previously about the image _always_ needing to be available, I wonder whether, when not `alt` text is provided, we should _not_ include the `img` at all. 

The `alt` text box in Gutenberg UI says

> Describe the purpose of the image(opens in a new tab)Leave empty if the image is purely decorative.

Purely decorative images do not need to be described in HTML as per this from the W3C

> Decorative images don’t add information to the content of a page. For example, the information provided by the image might already be given using adjacent text, or the image might be included to make the website more visually attractive. In these cases, a null (empty) alt text should be provided (alt="") so that they can be ignored by assistive technologies, such as screen readers

This is a super minor point, but it might be worth considering that if the user has not entered `alt` text we can assume the image is purely decorative and therefore not include the `img` at all. Whereas if `alt` text is provided we need to ensure the `img` tag is available.

Reply:
This is something we might want to consider. Given that this is behavior that predates this PR, in the interest of moving this PR along, I'd suggest that we might want to solve this in another PR. For now, I'll mark this conversation as resolved and we can open a separate issue for this.